frontend/src/metadata/MetadataPanel.svelte

```diff
@@ -219,6 +226,19 @@
 	<div id="slice-header" class="inline">
 		<h4>Slices</h4>
 		<div class="inline">
+			<div>
+				<Wrapper>
+					<IconButton on:click={() => (showSliceFinder = !showSliceFinder)}>
+						<Icon component={Svg} viewBox="0 0 24 24">
+							<path fill="black" d={mdiWifiPlus} />
+						</Icon>
+					</IconButton>
+					<Tooltip xPos="start">Create a Slice Finder</Tooltip>
```

Comment:
Change text to "Discover underperforming slices"

Reply:
Solved